include/zephyr/sys/byteorder.h

```diff
@@ -17,21 +17,21 @@
 #include <zephyr/toolchain.h>
 
 /* Internal helpers only used by the sys_* APIs further below */
-#define __bswap_16(x) ((uint16_t) ((((x) >> 8) & 0xff) | (((x) & 0xff) << 8)))
-#define __bswap_24(x) ((uint32_t) ((((x) >> 16) & 0xff) | \
+#define __z_bswap_16(x) ((uint16_t) ((((x) >> 8) & 0xff) | (((x) & 0xff) << 8)))
+#define __z_bswap_24(x) ((uint32_t) ((((x) >> 16) & 0xff) | \
```

Comment:
Instead of lot of changes, why not do it something like this so that we do not need to change the name of the macro
```
#if !defined(__bswap_16)
#define __bswap_16 ....
#endif  
```